pkg/controller/change/network/controller.go

```diff
@@ -158,6 +163,19 @@ func (r *Reconciler) createDeviceChanges(networkChange *networkchange.NetworkCha
 
 // canApplyChange returns a bool indicating whether the change can be applied
 func (r *Reconciler) canApplyChange(change *networkchange.NetworkChange) (bool, error) {
+	// First, check if the devices affected by the change are available
+	for _, deviceChange := range change.Changes {
```

Comment:
I think this is going to break the ability to have "config only" changes. 

It looks like in that case "device" will not be found and getProtocolState() will return Unknown and the overall result will be false?